File: leetcode/1-Two-Sum/answer.c
int* twoSum(int* nums, int numsSize, int target) {
    int* array = (int*)malloc(sizeof(int) * 2);
    int i;
    int j;
    for (i = 0; i < numsSize - 1; i++) {
        for (j = i + 1; j < numsSize; j++) {
            if (nums[i] + nums[j] == target) {
                array[0] = i;
                array[1] = j;
                break;
            }
        }
    }
    return array;
}

File: leetcode/1-Two-Sum/answer.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        num_dict = {}
        for index, num in enumerate(nums):
          diff = target - num
          if diff in num_dict:
            return [num_dict[diff], index]
          num_dict[num] = index
